extensions/wikia/Rail/styles/RailElement.scss

```diff
@@ -21,7 +17,7 @@ $color-rail-background: mix($color-page, $black, 90%);
 }
 
 @mixin wikia-rail-element() {
-    background-color: $color-rail-background;
+    background-color: $color-new-module-background;
```

Comment:
$color-module-background; already exists...
I know maybe it's not the best name for this color. It will be used in every module of newly created MOM's etc. Any ideas how the name would sound like?